.eslintrc.js

```diff
@@ -0,0 +1,20 @@
+module.exports = {
```

Comment:
Any reason you haven't used our standard eslint config here?
```suggestion
module.exports = {
  extends: [
    '@brickblock/eslint-config-base-ts',
  ],
}
```

Admittedly it is a config custom-tailored to Brickblock's opinions. But in the end every eslint config is opinionated so why create the extra work of having to maintain another one for `strong-config`?

Reply:
The goal is to go back to `@brickblock/eslint-config-base-ts`. To increase iteration speed and to have some room for experimentation, I used the plain rules here.

Let's backport to `@brickblock/eslint-config-base-ts` when we have a good setup.

Reply:
Interesting, what about `eslint-config-base-ts` slowed down your iteration speed exactly?

Reply:
Figuring out `eslint-config-base-ts` itself, seeing what needs overwrites and what not, ...

I know that rules can be overwritten if smth is off, but then I preferred no extend rather than a off one. If you care about this, let me know and I align the eslint setup in this MR

Reply:
Ok, doesn't need to be in this PR, but I do think it's best to use our full config sooner rather than later.

Feel free to resolve this after you create a GH issue to migrate to our config in one of the next PRs

Reply:
New issue: https://github.com/strong-config/node/issues/12

Are you interested in forming the ruleset for this?

Reply:
Thx, will resolve.

My suggestion for the initial ruleset is effectively [eslint-config-base-ts](https://git.brickblock.sh/core/eslint-config-base-ts) which simply uses [@typescript-eslint/recommended](https://github.com/typescript-eslint/typescript-eslint/blob/67537b863a8a98383796abb404224fdfc1c7b996/packages/eslint-plugin/src/configs/recommended.json)

If we want to customise the TypeScript rules a bit more, I think you'd probably be more qualified to do this by now as I haven't really done a lot of TypeScript development in the last 2 years.